rules/integrations/azure/initial_access_entra_id_protection_user_risk_detected.toml

```
@@ -0,0 +1,187 @@
+[metadata]
+creation_date = "2025/05/02"
+integration = ["azure"]
+maturity = "production"
+promotion = true
+updated_date = "2025/05/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies user risk detection events via Microsofts Entra ID Protection service. Entra ID Protection detects user risk
+activity such as anonymized IP addresses, unlikely travel, password spray, and more.
+"""
+false_positives = [
+    """
+    Users accessing their accounts from anonymized IP addresses, such as VPNs or Tor, may trigger this rule. If this is
+    expected behavior in your environment, consider adjusting the rule or adding exceptions for specific users or IP
+    ranges.
+    """,
+    """
+    Users who frequently travel or access their accounts from different geographic locations may trigger this rule due
+    to the unlikely travel detection mechanism. If this is expected behavior, consider adjusting the rule or adding
+    exceptions for specific users.
+    """,
+    """
+    Users who have recently changed their passwords may trigger this rule due to the password spray detection mechanism.
+    If this is expected behavior, consider adjusting the rule or adding exceptions for specific users.
+    """,
+]
+from = "now-9m"
```

I keep seeing this in our rules, and only just realized that the default interval for rules running is 5m so this setting means that there will be an additional 4m added to the lookback window every 5m. That large a lookback is going to cause some duplicate alerts as the same events are being evaluated across two rule execution windows. I would suggest changing this to `now-6m` so that's there's only 1m additional lookback added to the 5m default window. I'm going throught the AWS ruleset and changing these. 
```suggestion
from = "now-6m"
```